src/exportJarSteps/GenerateJarExecutor.ts

```diff
@@ -91,28 +102,39 @@ export class GenerateJarExecutor implements IExportJarStepExecutor {
                 });
                 const pickItems: IJarQuickPickItem[] = [];
                 const uriSet: Set<string> = new Set<string>();
-                for (const project of stepMetadata.projectList) {
+                const projectList: INodeData[] | undefined = stepMetadata.projectList;
+                if (!projectList || _.isEmpty(projectList)) {
```

Comment:
`isEmpty()` can handle with the null/undefined cases. Suggestion you can play with it in the official site: https://lodash.com/docs/4.17.15#isEmpty